src/qml/QFieldCloudPopup.qml

```diff
@@ -652,8 +711,6 @@ Popup {
         displayToast(qsTr('Local changes reverted'))
       else
         displayToast(qsTr('Failed to revert changes'), 'error')
-
-      return
```

Comment:
Sure about this one? There is another `displayToast` 3 lines below after the `if`?